src/os/osexec.go

```diff
@@ -0,0 +1,55 @@
+//go:build linux
```

Comment:
```suggestion
//go:build linux && !baremetal
```

Baremetal systems act like Linux, so you need to explicitly exclude these.

Reply:
But then most of the `os` package should have that set, right? 
The CI fails for other missing implementations such as `os.Getpid` and others.
I am note sure though why this is happening here and has not happened before?